Tiltfile

```diff
@@ -12,14 +12,18 @@
     * Other notes: does a `pip install` for package dependencies. Reinstalls dependencies iff they have changed.
 """
 
+k8s_resource_assembly_version(2)
 
-def get_username():
-    return str(local('whoami')).rstrip('\n')
+username = str(local('whoami')).rstrip('\n')
+
+
+def with_username(s):
```

Comment:
just in case you were unaware, another option is:
https://github.com/windmilleng/servantes/blob/master/Tiltfile#L126